package.json

```diff
@@ -119,5 +119,9 @@
     ],
     "ext": "js,css,hbs,json,ftl"
   },
-  "supportedLocales": "cak,cs,cy,da,de,el,en,en-CA,en-GB,es-AR,es-CL,es-ES,es-MX,fi,fr,fy-NL,gn,hu,kab,ia,id,it,ja,nb-NO,nl,nn-NO,pt-BR,pt-PT,ro,ru,sk,sl,sq,sv-SE,tr,uk,vi,zh-CN,zh-TW"
+  "supportedLocales": "cak,cs,cy,da,de,el,en,en-CA,en-GB,es-AR,es-CL,es-ES,es-MX,fi,fr,fy-NL,gn,hu,kab,ia,id,it,ja,nb-NO,nl,nn-NO,pt-BR,pt-PT,ro,ru,sk,sl,sq,sv-SE,tr,uk,vi,zh-CN,zh-TW",
+  "volta": {
+    "node": "18.12.1",
+    "npm": "8.19.3"
+  }
```

Comment:
I haven't seen any mention of that, but the two reasons I would expect are:

1. `engines` [doesn't necessarily resolve to a specific version](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#engines), i.e. you can specify a range of versions that the project is compatible with.
2. They serve different purposes, i.e. as a hint about compatibility for downstreams vs. aligning contributors on a specific version for consistent reproducible experiences.

I do agree that it would be nice to not have to update these in multiple places. My suggestion therefore would be to remove the `engines` field, since that's more aimed at downstreams - which we don't have, since we're not a library. What do you think?